Source/JavaScriptCore/Scripts/jsmin.py

```diff
@@ -24,40 +26,20 @@
 # THE SOFTWARE.
 
 
-import sys
-is_3 = sys.version_info >= (3, 0)
```

Comment:
I think stripping Python 2 support is the right call, but that's something more JavaScriptCore folks should verify